README.md

```diff
@@ -5,73 +5,41 @@ Licensed under the Apache License, Version 2.0 (the LICENSE-APACHE file) or
-  `elfmalloc`, `Bagpipe`s are used as "free lists" where ordering on the
-  insertion and removal of elements is not required.
-
-* `object-alloc` includes a number of traits currently used to define
-  object-specific allocators. Its traits are currently only implemented by the
-  allocators in `slab-alloc`, though we intend this to change over time.
-
-* `object-alloc-test` provides a number of tests that check for correctness of
-  an arbitrary object allocator (i.e. an allocator implementing one of the
-  traits in `object-alloc`).
-
-* `malloc-bind` is a crate that provides macros to transform any general
-  allocator that can store its own size and alignment-related metadata into a
-  full `malloc` implementation. The intent is to replace `elfc` with this more
-  generic solution in the future.
+# Allocators in Rust
+
+[![Build Status](https://travis-ci.org/ezrosent/allocators-rs.svg?branch=sysconf)](https://travis-ci.org/ezrosent/allocators-rs)
+[![Build status](https://ci.appveyor.com/api/projects/status/github/ezrosent/allocators-rs?svg=true)](https://ci.appveyor.com/project/ezrosent/allocators-rs)
+
+_Looking for elfmalloc in particular? It's [here](https://github.com/ezrosent/allocators-rs/blob/master/elfmalloc)._
+
+This repository encompasses a number of different crates. Some are general
+memory allocators, some are object allocators (allocators which allocate a
+specific type of object), and some are utility crates providing various
+features needed to implement these allocators.
+
+The [`info`](https://github.com/ezrosent/allocators-rs/blob/master/info) directory contains more detailed information, including performance
+measurements. Aside from `info`, all top-level directories are Rust crates.
+More detailed information on each crate can be found in the crate's `README.md`.
+
+## Crates
+
+### Allocators
+
+| Crate | Description |
+|-------|-------------|
+| [`elfmalloc`](https://github.com/ezrosent/allocators-rs/blob/master/elfmalloc) | A general-purpose multi-threaded allocator providing both Rust `Alloc` and C `malloc` APIs |
+| [`slab-alloc`](https://github.com/ezrosent/allocators-rs/blob/master/slab-alloc) | An object-specific slab allocator in the same tradition as the original [slab allocator](https://www.usenix.org/legacy/publications/library/proceedings/bos94/full_papers/bonwick.a) by Jeff Bonwick |
+| [`bsalloc`](https://github.com/ezrosent/allocators-rs/blob/master/bsalloc) | A simple general-purpose "bootstrapping" allocator used in the implementation of other allocators |
+| [`mmap-alloc`](https://github.com/ezrosent/allocators-rs/blob/master/mmap-alloc) | An `Alloc` API which is a thin wrapper around `mmap` (on Unix) or `VirtualAlloc` (on Windows) |
+
+### Utility crates
+
+| Crate | Description |
+|-------|-------------|
+| [`bagpipe`](https://github.com/ezrosent/allocators-rs/blob/master/bagpipe) | A fast, concurrent weakly-ordered bag data structure |
```

Comment:
It also provides some concurrent queues as well.
It may be worth linking to the [bagpipe design](https://github.com/ezrosent/allocators-rs/blob/master/info/bagpipes.md) there